From 32cb9a8326b579678dec55ed571e27e682d0abd5 Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Fri, 26 Feb 2016 15:45:30 +0000
Subject: [PATCH] udev rules: add kvm rule

Based on commit e8c942d0df396e3c756a7261a8bffcb8c9847a1a.

Make /dev/kvm available to members of the kvm group. This improves
security since it avoids having to be root to make use of kvm.

Signed-off-by: Vu Tran <vu.tran@windriver.com>
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>

Update fuzzy patch.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 rules.d/99-kvm.rules | 1 +
 rules.d/meson.build  | 1 +
 2 files changed, 2 insertions(+)
 create mode 100644 rules.d/99-kvm.rules

diff --git a/rules.d/99-kvm.rules b/rules.d/99-kvm.rules
new file mode 100644
index 0000000000..c2f7317aac
--- /dev/null
+++ b/rules.d/99-kvm.rules
@@ -0,0 +1 @@
+KERNEL=="kvm", GROUP="kvm", MODE="0660"
diff --git a/rules.d/meson.build b/rules.d/meson.build
index b4640ab9d4..8387996d62 100644
--- a/rules.d/meson.build
+++ b/rules.d/meson.build
@@ -22,6 +22,7 @@ rules = files('''
         75-probe_mtd.rules
         78-sound-card.rules
         80-net-setup-link.rules
+        99-kvm.rules
 '''.split())
 
 if conf.get('HAVE_KMOD') == 1
